superchain/configs/sepolia-dev-0/superchain.yaml

```diff
@@ -0,0 +1,8 @@
+name: Sepolia Dev 0
+l1:
+  chain_id: 11155111
+  public_rpc: https://ethereum-sepolia-rpc.allthatnode.com
+  explorer: https://sepolia.etherscan.io
+
+protocol_versions_addr: null # todo
+superchain_config_addr: null # todo
```

Comment:
do you need to fill in these TODOs?

Reply:
I think so -- I'm not sure we have these superchain contracts deployed on sepolia yet?

Reply:
we just deployed them
```suggestion
superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"
```